Joomla4+

gug2 opened the pull request.

gug2 committed:
Адаптация под архитектуру Joomla 4, Joomla5+.

gug2 committed:
Исправление локализаций

gug2 committed:
Исправление локализации 2

gug2 committed:
Исправление отображения поля материалов Joomla

gug2 committed:
Avoid 'undefined array key' warning

sergeytolkachyov commented on services/provider.php:
```diff
@@ -0,0 +1,44 @@
+use Joomla\Event\DispatcherInterface;
+use Joomla\Plugin\Jshopping\Wt_jshopping_content_to_com_content\Extension\Wt_jshopping_content_to_com_content;
+
+return new class implements ServiceProviderInterface {
```
В Joomla 4.3+ провайдеры вот так со скобками идут. 
```suggestion
return new class () implements ServiceProviderInterface {
```

sergeytolkachyov reviewed (approved):
Вроде всё хорошо 👍

sergeytolkachyov committed:
Update services/provider.php

sergeytolkachyov committed:
Update plg_jshopping_wt_jshopping_content_to_com_content.sys.ini

- Использовал стандартное сообщение "Joomla 5 ready"
- вырезал iframe с Youtube, так как всё равно его джумла режет из системных сообщений

sergeytolkachyov committed:
Update plg_jshopping_wt_jshopping_content_to_com_content.sys.ini

- Вырезали ютуб видео
- стандартное сообщение "Joomla 5 ready"

sergeytolkachyov committed:
Update wt_jshopping_content_to_com_content.xml

- udpate server added
- changelog server added
